pysilcam/postprocess.py

```diff
@@ -311,29 +316,28 @@ def __init__(self, settings):
         self.dias = None
 
     def update_from_stats(self, stats, timestamp):
-        '''Update size distribution from stats'''
+        """Update size distribution from stats"""
         dias, vd = vd_from_stats(stats, self.settings)
         self.dias = dias
 
-        #Add the new data
+        # Add the new data
         self.times.append(timestamp)
         self.vdlist.append(vd)
 
-        #Remove data until we are within window size
+        # Remove data until we are within window size
         while (timestamp - self.times[0]).seconds > self.window_size:
             self.times.pop(0)
             self.vdlist.pop(0)
 
-        #Calculate time-integrated volume distribution
-        if len(self.vdlist)>1:
+        # Calculate time-integrated volume distribution
+        if len(self.vdlist) > 1:
             self.vd_mean = np.nanmean(self.vdlist, axis=0)
         else:
             self.vd_mean = self.vdlist[0]
 
 
-def montage_maker(roifiles, roidir, pixel_size, msize=2048, brightness=255,
-        tightpack=False, eyecandy=True):
-    '''
+def montage_maker(roifiles, roidir, pixel_size, msize=2048, brightness=255, tightpack=False, eyecandy=True):
```

Comment:
C901: 'montage_maker' is too complex (13)  <!-- Automatically posted by Lintly -->